pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -70,15 +70,17 @@ type HelmCheck struct {
 }
 
 type checkConfig struct {
-	CollectEvents                  bool `yaml:"collect_events"`
-	ExtraSyncTimeoutSeconds        int  `yaml:"extra_sync_timeout_seconds"`
-	InformersResyncIntervalMinutes int  `yaml:"informers_resync_interval_minutes"`
+	CollectEvents                  bool              `yaml:"collect_events"`
+	ConfigParamsAsTags             map[string]string `yaml:"config_params_as_tags"`
```

Comment:
Not strong opinion either. In the Helm codebase "values" is the name for the defaults map (https://github.com/helm/helm/blob/bed23120b048977d67c929cec915e1d6e0887c5d/pkg/chart/chart.go#L48) and "config" is the name for the overrides: https://github.com/helm/helm/blob/bed23120b048977d67c929cec915e1d6e0887c5d/pkg/release/release.go#L31

Reply:
sounds good then, thanks for the links!

Reply:
I've changed my opinion. I think your initial suggestion is better because these params are defined in a file called "values.yaml", so I think for users `helm_values` will be more intuitive. I added a new commit with this.